tests/validator/bl/NextflowConfigTests.groovy

```diff
@@ -0,0 +1,54 @@
+package validator.bl
+
+import org.junit.Before
+import org.junit.Rule
+import org.junit.contrib.java.lang.system.EnvironmentVariables
+import org.junit.rules.TemporaryFolder
+
+import groovy.util.ConfigObject
+import java.nio.file.Path
+
+import validator.bl.BLConfigBuilder
+
+
+abstract class NextflowConfigTests {
+    protected BLConfigBuilder builder
+    protected ConfigObject inconfig, expected
+
+    @Rule
+    public final EnvironmentVariables envvars = new EnvironmentVariables()
+
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder()
+
+    @Before
+    public void setup() {
+        this.builder = new BLConfigBuilder()
+        this.builder.setBaseDir(get_projectDir())
+
+        this.inconfig = new ConfigObject()
+        this.expected = new ConfigObject()
+    }
```

Comment:
Another note - the builder needs to be created per-test (using `@Before`) because it internally memoizes the result of expanding each named config file. Trying to share a builder created with `@BeforeClass` causes interference between tests.